docs/AddingNewProjects.md

```diff
@@ -51,4 +51,4 @@ while the second (the one with condition `'$(BuildMainlyReferenceProviders)' ==
 3. Add your project name to the lists in `src\Framework\test\TestData.cs`. This is not strictly necessary for the project to work but there is a test on CI that will fail if this is not done. Make sure to include your project in a way that maintains alphabetical ordering.
 
 ## Manually saving solution and solution filter files
-VS is pretty good at keeping the files up to date and organized correctly. It will also prompt you if it finds an error and, in most cases, offer a solution to fix the issue. Sometimes just saving the file will trigger VS to resolve any issues automatically. However, if you would like to add a new solution filter file or update one manually you can find a tutorial link [here](https://docs.microsoft.com/en-us/visualstudio/ide/filtered-solutions?view=vs-2022).
+VS is pretty good at keeping the files up to date and organized correctly. It will also prompt you if it finds an error and, in most cases, offer a solution to fix the issue. Sometimes just saving the file will trigger VS to resolve any issues automatically. However, if you would like to add a new solution filter file or update one manually you can find a tutorial link [here](https://learn.microsoft.com/en-us/visualstudio/ide/filtered-solutions?view=vs-2022).
```

Comment:
I'm not going repeat my comments about versions everywhere. The URIs all redirect to the latest view and that'll avoid having to update the docs whenever a new VS or .NET release happens.

Reply:
I took a look. I think I was able to version stuff, and I checked to make sure they all still work.